main_test.go

```diff
@@ -271,3 +271,86 @@ func TestTransferNonFungibleTokensToTreasuryActions(t *testing.T) {
 		assert.Contains(otu.T, ownedNFTIds, uint64(0))
 	})
 }
+
+func TestAddSignerAction(t *testing.T) {
+	var addSignerActionUUID uint64
+
+	otu := NewOverflowTest(t)
+	otu.SetupTreasury("signer1", Signers, uint64(DefaultThreshold))
```

Comment:
Small nit, probably would be better to keep the convention `treasuryOwner` instead of `singer1`, it helps line 281 read a little better as well ("treasuryOwner proposed adding signer6" instead of "singer1 proposed adding signer6")